src/EducationService.Models.Dto/Requests/User/FindUsersFilter.cs

```diff
@@ -0,0 +1,17 @@
+﻿using LT.DigitalOffice.Kernel.Requests;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.EducationService.Models.Dto.Requests.User;
+
+public record FindUsersFilter : BaseFindFilter
+{
+  [FromQuery(Name = "educationFormId")]
```

Comment:
We usually write 'education**f**orm**i**d'

Reply:
Done